style.css

```diff
@@ -0,0 +1,25 @@
+
+.container{
+    margin: 0 auto;
+    max-width: 1325px;    
+}
+
+
+header {
+    display: flex;
+    justify-content: space-between;
+    padding: 30px 0px 0px 0px;
```

Comment:
It looks like you want to set only the top padding so you don't need to use this shorthand, you can do this instead: `padding-top: 30px`.

Also, it looks like you want to add a gap between the header section and the
 top of the page, my recommendation, in this case, is use margin instead of padding, ex: `margin-top: 30px`

Padding is used to set space between the section and the content that is inside the container, in this case, you want to separate the header section.

Reply:
I just set the margin-top for this section. By the way, it seems I applied the changes after I git push the header branch because the local file doesn't have the padding attribute anymore. 

Thank you for the feedback.